tests/pytorch_tests/model_tests/feature_models/conv2d_replacement_test.py

```diff
@@ -65,7 +66,7 @@ def create_feature_network(self, input_shape):
     def compare(self, quantized_models, float_model, input_x=None, quantization_info=None):
         quantized_model = quantized_models.get('no_quantization')
         if self.experimental_exporter:
-            self.unit_test.assertTrue(isinstance(quantized_model.conv1, QuantizedLayerWrapper))
+            self.unit_test.assertTrue(isinstance(quantized_model.conv1, PytorchQuantizationWrapper))
             self.unit_test.assertTrue(isinstance(quantized_model.conv1.layer, torch.nn.Conv2d))
             self.unit_test.assertTrue(torch.all(quantized_model.conv1.layer.weight == 1))
         else:
```

Comment:
Please remove the "else" part (old testing part that tested the previously exported model)